packages/sanity/src/core/studio/components/navbar/search/contexts/search/SearchProvider.tsx

```diff
@@ -24,15 +25,29 @@ import {initialSearchState, searchReducer} from './reducer'
 interface SearchProviderProps {
   children?: ReactNode
   fullscreen?: boolean
+  /**
+   * if provided, then it means that the search is being done from within a specific perspective
+   */
+  perspective?: string
```

Comment:
Did you consider an array here? Since we use the bundlePerspective for document lists I'm wondering if it would be expected that search results matches the current bundle perspectives.

Reply:
Hmm this is actually where right now I'm indicating that we /shouldn't/ use the bundlePerspectives. If we put "raw" here, then it means that the search will ignore the bundlePerspectives and simply look through all existing releases.

Re-reading my description I feel it's wrong. All that I do right now in the search aspect is check if it's raw, if it's raw then it does what I mentioned, but if it has nothing, then it will always search through the existingReleases (vs actually searching focused in a particular perspective).

I'll have a look, I think it might be simple to not make this comment be a lie (as in, if the perspective is actually sent in as something else, then search only within that perspective). I'll come back to you on this

Reply:
This has quite a few large impacting changes on the code 🤔 
It'll take a little bit longer to get this to work as intended I think, I'll timebox this until tomorrow before the sync

Reply:
Here we go, done here [refactor(sanity): update to array prop vs string](https://github.com/sanity-io/sanity/pull/7815/commits/fe34b34a6299959a7bd0fd6e9f9c7f9ea76e361d) :) 

Thanks for the suggestion!